src/main/java/no/sikt/sws/models/opensearch/OpenSearchIndexDto.java

```diff
@@ -2,8 +2,9 @@
 
 import com.fasterxml.jackson.annotation.JsonProperty;
 import com.fasterxml.jackson.databind.JsonNode;
+import no.sikt.sws.models.gateway.Dto;
 
-public class OpenSearchIndexDto {
+public class OpenSearchIndexDto implements Dto {
```

Comment:
Here you implement Dto which is located in ../gateway. You should move Dto to be directly under models so that it makes sense to use it in "internal", "gateway", and "opensearch"-packages